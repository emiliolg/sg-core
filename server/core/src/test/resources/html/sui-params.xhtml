<html sui-view="tekgenesis.service.html.Views" sui-params="a, b : String, c : tekgenesis.samples.service.Product" prefix="og: http://ogp.me/ns#">
    <head>
    </head>
    <body>
    </body>
</html>
