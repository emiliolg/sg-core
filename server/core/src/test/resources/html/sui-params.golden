<!DOCTYPE html>
<html prefix="og: http://ogp.me/ns#" sui-scope="/html/sui-params.xhtml/0">
    <head>
    </head>
    <body>
    
<script class="suiScopesCache">
//<![CDATA[
var suiScopesCache = {"/html/sui-params.xhtml/0":{"a":"Value 'a'","b":"Value 'b'","c":{"productId":"product.id","model":"Some model","description":"Some description with \"quoted\" words.","price":89.99,"created":"2014-01-31T03:00:00Z"}}};
//]]>
</script>
</body>
</html>